src/quicer.erl

```diff
@@ -727,11 +727,20 @@ do_recv(Stream, Count, Buff) ->
         {error, dgram_send_error, atom_reason()}.
 send_dgram(Conn, Data) ->
   case quicer_nif:send_dgram(Conn, Data, _IsSync = 1) of
-    %% @todo make ref
+    %% @todo we need find tuned event mask
     {ok, _Len} = OK ->
       receive
-        {quic, send_dgram_completed, Conn} ->
-          OK
+        {quic, dgram_send_state, Conn, #{state := ?QUIC_DATAGRAM_SEND_SENT}} ->
```

Comment:
these are not nice, we could intro some dgram event mask for caller to skip uninterested events. also intro `async_send_dgram`.